x-pack/plugins/ml/server/models/job_validation/messages.json

```diff
@@ -35,57 +35,70 @@
   },
   "categorization_filters_valid": {
     "status": "SUCCESS",
-    "text": "Categorization filters"
+    "text": "Categorization filters checks passed.",
+    "url": "https://www.elastic.co/guide/en/x-pack/{{version}}/ml-configuring-categories.html#ml-configuring-categories"
   },
   "categorization_filters_invalid": {
     "status": "ERROR",
     "text": "The categorization filters are invalid."
```

Comment:
If we can't provide more details about why the categorization filters are invalid, would it be helpful to at least point them here for more information about that property?: https://www.elastic.co/guide/en/elasticsearch/reference/current/ml-job-resource.html#ml-analysisconfig

Reply:
Does this message get displayed when both `categorization_filters` and a `categorization_analyzer` are configured?  If so then the message could be "Categorization filters are not permitted with a categorization analyzer.  Instead add a `char_filter` within the `categorization_analyzer`."  It should still link to the docs as well, as this whole area of the config is complex.

Reply:
Here's the current code that does the validation: https://github.com/elastic/kibana/blob/master/x-pack/plugins/ml/common/util/job_utils.js#L264

At the moment we check that every filter is a valid regular expression and when filters are set that `job.analysis_config.categorization_field_name` must also be set. We could extend this in the future, but to match the current validation, I suggest to change the text for now to:

> The categorization filters configuration is invalid. Make sure filters are valid regular expressions and "categorization_field_name" is set.

Reply:
@droberts195 I added a note to the job validation meta issue about `categorization_filters/categorization_analyzer` https://github.com/elastic/kibana/issues/18368